addons/digest/data/digest_data.xml

```diff
@@ -416,6 +416,13 @@
                         <span style="color: #8f8f8f;">Unsubscribe</span>
                     </a>
                 </t>
+                <t t-elif="mailing_off_token">
+                    –
+                    <a t-attf-href="/mass_mailing/turnoff_report?token=#{mailing_off_token}"
```

Comment:
you use a route defined in mass mailing in the digest module... digest doesn't depends on mass mailing so you can not do that, you should inherit the template in mass mailing